.github/copilot-instructions.md

```
@@ -2,243 +2,224 @@
-      meshRef.current.rotation.y += delta;
-    }
-  });
-  
-  return (
-    <mesh ref={meshRef}>
-      <boxGeometry args={[1, 1, 1]} />
-      <meshStandardMaterial color="cyan" />
-    </mesh>
-  );
-}
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)